modules/client-configuration/pages/registration-bootstrap.adoc

```diff
@@ -99,7 +107,7 @@ cp bootstrap.sh bootstrap-sles15.sh
 ----
 . Open [path]``bootstrap-sles12.sh`` for modification.
 Scroll down until you can see the text shown below.
-If ``exit 1`` exists in the file, comment it out by typing a hash or pound sign (``#``) at the beginning of the line.
+If ``exit 1`` exists in the file, comment it out by typing a hash or pound sign at the beginning of the line.
```

Comment:
So I guess we have `pound sign` because: 

> For some people in the US, "pound sign" refers to the symbol # (number sign).

However for non-native (at least for me) pound sign is `£`. So if you do this change, for me it means I can comment with `#` or `£` which is not correct.

Reply:
heh, I just took it out because it was messy and it was breaking my code highlighting. I'll add it back (personally I thought "hash or pound" covered it, but I can see the point).